optuna/samplers/__init__.py

```diff
@@ -1,7 +1,7 @@
 from optuna.samplers._base import BaseSampler
 from optuna.samplers._cmaes import CmaEsSampler
 from optuna.samplers._grid import GridSampler
-from optuna.samplers._nsga2 import NSGAIISampler
+from optuna.samplers._nsga2.sampler import NSGAIISampler
```

Comment:
How about adding this line in `optuna/samplers/_nsga2/__init__.py`? Then `NSGAIISampler` can be imported as before.